NEWS.md

```diff
@@ -88,6 +90,8 @@
 
 * `routine_registration_linter()` for identifying native routines that don't use registration (`useDynLib` in the `NAMESPACE`; @MichaelChirico)
 
+*  `indentation_linter()`: see above in 'Changes to defaults'
```

Comment:
I figure it's good to add this under "new linters" in case there are any readers just focus on that when scanning the NEWS, WDYT?

Reply:
Makes more sense - good idea. (I.e. " New `indentation_linter()`" is part of the default linters under "Changes to defaults" and the rest under "New linters").